example/rocketmq_v1alpha1_broker_cr.yaml

```diff
@@ -27,6 +27,10 @@ spec:
   replicationMode: ASYNC
   # replicaPerGroup is the number of each broker cluster
   replicaPerGroup: 1
+  # nodeAffinityKey|nodeAffinityKey|nodeAffinityValues used to schedule specified nodes
```

Comment:
How should the users write the CR config yaml if they do not need NodeAffinity ?